pkg/controller/unreachable/unreachable_controller.go

```diff
@@ -170,6 +170,7 @@ func (r *ReconcileRemoteMachineSet) Reconcile(request reconcile.Request) (reconc
 	// Attempt to connect to the remote cluster using the preferred API URL.
 	_, primaryErr := remoteClientBuilder.UsePrimaryAPIURL().Build()
 	if primaryErr != nil {
+		cdLog.WithError(primaryErr).Debug("cluster is not reachable via primary URL")
```

Comment:
Isn't this already covered by the following two log entries?
```
cdLog.WithError(primaryErr).Info("unable to create remote API client using API URL override")
```
```
cdLog.WithError(primaryErr).Warn("unable to create remote API client, marking cluster unreachable")
```

Reply:
Yes, that should be removed